File: java/src/n01_oop_vererbung/Azubi.java
package n01_oop_vererbung;

public class Azubi extends Mitarbeiter {
    public int lehrjahr;

    public Azubi(String name, String gebOrt, String gebDat, int personalNummer, boolean fuehrungskraft, double gehalt, int lehrjahr) {
        super(name, gebOrt, gebDat, personalNummer, fuehrungskraft, gehalt);
        this.lehrjahr = lehrjahr;
    }
}

File: dart/src/n01_oop_vererbung/azubi.dart
import 'mitarbeiter.dart';

class Azubi extends Mitarbeiter {
  Azubi(super.name, super.gebOrt, super.gebDat, super.personalNummer, super.fuehrungskraft, super.gehalt, this.lehrjahr);

  final int lehrjahr;
}
